public/app/core/components/TransformersUI/spatial/spatialTransformer.ts

```diff
@@ -49,6 +49,7 @@ async function doSetGeometry(frames: DataFrame[], options: SpatialTransformOptio
             }
             return {
               ...frame,
+              length: 1,
               fields: [
                 {
                   ...info.field,
```

Comment:
Those spatial transformers would use some docs.

Reply:
> Those spatial transformers would use some docs.
Currently alpha and under heavy development :)